### PR TITLE
Add a etcd java sdk by ibm

### PR DESCRIPTION
**Pull request by pubxun**

### CODE REVIEW

**Review by spzala — APPROVED** [review]
lgtm
Need +1 from other maintainers @gyuho @jingyih @ptabor

**Comment by jingyih** [comment]
lgtm

**Comment by ptabor** [comment]
lgtm

**Comment by spzala** [comment]
Thanks @ptabor

**Pull request merged**
